runHarness/Services/KubernetesService.pm

```diff
@@ -101,6 +101,41 @@ override 'isStopped' => sub {
 	return !$self->host->kubernetesDoPodsExist("type=$serviceType", $namespace );
 };
 
+sub expandK8sCpu {
+	my ($self, $cpuString, $expansionFactor) = @_;
+	
```

Comment:
Should there be some input validation on the expansionFactor string for this cpu and in the mem case? (numerical, >= 0)

Reply:
I added validation for the value of the limitsExpansionPct in AuctionAppInstance::checkConfig.  I tested the validation and it works.